tests/errors/semantic/blocking/SORT_LIST_OPTIONAL_PARAM.py

```diff
@@ -0,0 +1,6 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+
+a = [1, 2]
+a.sort(reverse=True, key=len(a))
```

Comment:
It would be better to use an example that runs in Python. This code raises:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-923795b47a06> in <module>
----> 1 a.sort(reverse=True, key=len(a))

TypeError: 'int' object is not callable

```

Reply:
fixed c7b4cae7f696ddc73b85fa805cbdebb02a362c4e